apps/L1G/tinder/tinder-frontend/jest.setup.js

```
@@ -0,0 +1,6 @@
+window.HTMLElement.prototype.scrollIntoView = jest.fn();
```

ESLint

[error] 'jest' is not defined.

(no-undef)